releasenotes/notes/noreno-label-consistency-f6e91c6270651040.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    PRs that don't include release notes must use the changelog/no-changelog
```

Comment:
I don't think a release note is necessary here. Their purpose is to advertise the changes on the agent, not on our dev process. I think you should remove it and add a `changelog/no-changelog` tag 😄

Reply:
Done.